test/CowAmmModule/CreateAmm.sol

```diff
@@ -0,0 +1,82 @@
+// SPDX-License-Identifier: GPL-3.0
+pragma solidity >=0.8.0 <0.9.0;
+
+import {CowAmmModuleTestHarness, CowAmmModule, ConstantProduct} from "./CowAmmModuleTestHarness.sol";
+import {FallbackManager} from "lib/composable-cow/lib/safe/contracts/Safe.sol";
+
+abstract contract CreateAmmTest is CowAmmModuleTestHarness {
+    function testCreateAmm() public {
+        setUpDefaultSafe();
+
+        ConstantProduct.Data memory ammData = getDefaultData();
+        bytes32 domainSeparator = settlement.domainSeparator();
+
+        vm.prank(address(safe));
+
+        // Verify `ChangedFallbackHandler` and should be set to `eHandler`
+        // We do this to ensure that the fallback handler is set to the expected value
+        // as observing the handler directly is not possible
+        vm.expectEmit(true, true, false, false);
+        emit FallbackManager.ChangedFallbackHandler(address(eHandler));
+        vm.expectEmit(true, true, true, false);
+        emit CowAmmModule.CowAmmCreated(safe, token0, token1, bytes32(0));
```

Comment:
Why don't we want to test the last parameter (order hash) as well?

In general, in all instances of `expectEmit` in this PR I'd expect that we don't want to explicitly say which topic to check and most time we just want to use `vm.expectEmit()` with no arguments. Am I missing something?